src/website/views/components/activity.pug

```diff
@@ -0,0 +1,1105 @@
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
+                  <li>
+                    <div>
+                      <div class="a-icon -text--success">
+                        <svg>
+                          <use xlink:href="#icon-check-alt"></use>
+                        </svg>
+                      </div>
+                      Instance successfully deployed
+                    </div>
+                    <div class="-time">1 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--success">
+                        <svg>
+                          <use xlink:href="#icon-check-alt"></use>
+                        </svg>
+                      </div>
+                      Machine active-directory-vbo8o-1 successfully deployed
+                    </div>
+                    <div class="-time">2 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--muted">
+                        <svg>
+                          <use xlink:href="#icon-circle-plus-outline"></use>
+                        </svg>
+                      </div>
+                      Executing install:Active Directory in machine active-directory-vbo8o-1
+                    </div>
+                    <div class="-time">2 minute ago</div>
+                  </li>
+                  <li>
+                    <div>
+                      <div class="a-icon -text--muted">
+                        <svg>
+                          <use xlink:href="#icon-circle-plus-outline"></use>
+                        </svg>
+                      </div>
+                      Updating machine IPs
+                    </div>
+                    <div class="-time">3 minute ago</div>
+                  </li>
+                </ul>
+              </div>
+              <div class="m-activity__end-footer">04/22/2019</div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="m-activity__day">
+        <div class="m-activity__content">
+          <div class="m-activity__item">
+            <div class="m-activity__start">
+              <div>
+                <div class="a-icon -text--muted">
+                  <svg>
+                    <use xlink:href="#icon-circle-play-outline"></use>
+                  </svg>
+                </div>
+                <div class="a-avatar"><img src="../../assets/images/avatar.jpg" alt="avatar"></div>
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
```

Comment:
@davidojedacl is this needed for any purpose?  If don't, please remove it.

Reply:
done